pyproject.toml

```diff
@@ -19,29 +19,31 @@ classifiers = [
     "Programming Language :: Python",
 ]
 dependencies = [
-    "asdf",
-    "astropy",
-    "astroquery",
-    "bokeh>=3",
-    "crds",
-    "cryptography",
-    "django",
-    "inflection",
-    "jinja2",
-    "jsonschema",
-    "jwst",
-    "jwst_reffiles",
-    "matplotlib",
-    "numpy",
-    "numpydoc",
-    "pandas",
-    "psycopg2-binary",
-    "pysiaf",
-    "pyvo",
-    "scipy",
-    "sqlalchemy",
-    "stdatamodels",
-    "wtforms",
+    "asdf>=3.1.0,<4",
+    "astropy>=6.0,<7",
+    "astroquery>=0.4.7,<0.5.0",
+    "beautifulsoup4>=4.12.3,<5",
+    "bokeh>=3.4.0,<4",
+    "celery>=5.3.6,<6",
+    "crds>=11.17.19,<12",
+    "django>=5.0.3,<6",
+    "gunicorn",
```

Comment:
make sure to provide an upper pin for this as well; I don't use `gunicorn` so I don't know what their conventions are